include/zephyr/posix/posix_types.h

```diff
@@ -84,11 +84,6 @@ typedef unsigned long useconds_t;
 #endif
 
 /* time related attributes */
-#if !defined(CONFIG_NEWLIB_LIBC) && !defined(CONFIG_ARCMWDT_LIBC)
```

Comment:
This change doesn't seem to be relevant to making code work with ARCMWDT? It seems to be explicitly skipping this block when using that toolchain, after all?

Reply:
Yeah, so that's a separate commit. It removes a block of code that became unreachable and redundant (see lines 20 to 24 of the same file). Still, if you believe this change is wrong, I'll drop the commit.

Reply:
Thanks for the explanation. Looks good to me.